### ISSUE TITLE
type of 'create' object in docs not present

### ISSUE THREAD

**Issue by pdeva** [jOOQ/jOOQ#9003]
### Use case:

The documentation here uses the object `create`.

https://www.jooq.org/doc/3.11/manual/sql-execution/transaction-management/

The 'type' of `create` is not specified anywhere, and the object is not shown being initialized making it very confusing to follow the docs.

### Possible solution you'd like to see:

please list the 'type' of the objects being used

**Comment by knutwannheden** [comment]
Thanks for reporting this.

It is in fact documented here, but a bit hidden: https://www.jooq.org/doc/3.11/manual/getting-started/the-manual/.

We will try to think of a way to communicate this more clearly in the future.

**Comment by pdeva** [comment]
in the page you mentioned:
you use 'create' before you describe it in the section below there, which is very confusing.

**Comment by knutwannheden** [comment]
Yes, that can probably be improved as well. We would actually like to hyperlink all references in the examples to the corresponding Javadoc. I think this should improve things quite a bit.
